crates/extensions/aws_mapper_ext/src/converter.rs

```diff
@@ -37,44 +37,51 @@ impl AwsConverter {
     }
 
     fn try_convert(&mut self, input: &MqttMessage) -> Result<Vec<MqttMessage>, ConversionError> {
-        let default_timestamp = self.add_timestamp.then(|| self.clock.now());
-
-        // serialize with ThinEdgeJson for measurements, for alarms and events just add the timestamp
-        let payload = if input.topic.name.starts_with("tedge/measurements") {
-            let mut serializer = ThinEdgeJsonSerializer::new_with_timestamp(default_timestamp);
-            tedge_api::parser::parse_str(input.payload_str()?, &mut serializer)?;
-
-            serializer.into_string()?
-        } else if input.topic.name.starts_with("tedge/events")
-            || input.topic.name.starts_with("tedge/alarms")
-            || input.topic.name.starts_with("tedge/health")
+        if input.topic.name.eq("tedge/health/mosquitto-az-bridge")
+            || input.topic.name.eq("tedge/health/mosquitto-aws-bridge")
+            || input.topic.name.eq("tedge/health/mosquitto-c8y-bridge")
```

Comment:
Better to avoid exact matches on a the *current* list of bridge, as adding a new one would break again this mapper.

Reply:
fixed, added a function to check the bridge dynamically.